signer/src/transaction_coordinator.rs

```diff
@@ -2137,13 +2137,14 @@ where
 
         // Get the current sBTC limits (caps).
         let sbtc_limits = self.context.state().get_current_limits();
+        let signature_threshold = config.signer.bootstrap_signatures_required;
```

Comment:
As with the presign one, this may not be the correct one: it should probably a mix of each deposit aggregate key shares threshold and the registry (or maybe the config?) one for the stacks side. I'm not sure if we can safely roll with the config one in this case, eg if we lower the threshold (say, from 11 to 10) and we have a deposit with 10 for and 4 against spent to the old aggregate key (the one with 11 as threshold), using the new threshold we may try to include it, but we would never manage to sign for it. And similarly, if we increase the threshold we may cut off valid deposits using spent to a lower threshold aggregate key.